charts/fleet/100.2.0+up0.5.0/Chart.yaml.orig

```diff
@@ -0,0 +1,20 @@
+annotations:
```

Comment:
There should be no `*.orig` files to check in. Means the patch didn't apply cleanly. The content of this file shouldn't need to exist.